extensions/adobe/experience/offer-management/offer.schema.json

```diff
@@ -0,0 +1,89 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://ns.adobe.com/experience/offer-management/offer",
+  "meta:extensible": "true",
+  "type": "object",
+  "title": "Offer",
+  "description": "Adobe Experience Cloud Offer",
+  "definitions": {
+    "offer": {
+      "properties": {
+        "@id": {
+          "type": "string",
+          "format": "uri",
+          "title": "ID",
+          "description": "The unique identifier of the offer"
+        },
+        "xdm:name": {
+          "type": "string",
+          "title": "Name",
+          "description": "Offer name"
+        },
+        "xdm:status": {
+          "type": "string",
+          "title": "Status",
+          "description": "Offer Status",
+          "enum": [
+            "draft",
+            "pending",
+            "rejected",
+            "approved",
+            "inUse",
+            "archived"
+          ],
+          "meta:enum": {
+            "draft": "Draft",
+            "pending": "Pending",
+            "rejected": "Rejected",
+            "approved": "Approved",
+            "inUse": "In Use",
+            "archived": "Archived"
+          }
+        },
+        "xdm:representations": {
+          "type": "array",
+          "items": {
+            "$ref":
+              "https://ns.adobe.com/experience/offer-management/offer-representation"
+          },
+          "title": "Representations",
+          "description": "Offer representations for different placements"
+        },
+        "xdm:customMetadata": {
```

Comment:
See above. We can handle customMetadata through url-value pairs. There's already an extensibility model in XDM.

Reply:
This is meant to be an end-user supplied data.

Reply:
To elaborate on this. We needed a mechanism where an API client (and without modifying the XDM schema by extension) has means to provide additional named values in the JSON payload.